avaframe/com1DFA/com1DFA.py

```diff
@@ -1415,6 +1422,8 @@ def computeEulerTimeStep(cfg, particles, fields, dt, dem, Tcpu, frictType):
     startTime = time.time()
     log.debug('update Fields C')
     # particles, fields = updateFields(cfg, particles, force, dem, fields)
+
+    #if flowDepthOption!=1 or viscOption !=2 :
```

Comment:
what does this comment mean?

Reply:
nothing, thats an error

Reply:
@matthiasto do we have to put the function 'computeFlowDepthSPH' under the startTime ?